pkg/metrics/metrics.go

```diff
@@ -47,6 +47,14 @@ var (
 		},
 		[]string{"mode"},
 	)
+	// PersistentVolumeSizeBytes is used to collect the size of the local volumes discovered.
+	PersistentVolumeSizeBytes = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Subsystem: LocalVolumeProvisionerSubsystem,
+			Name:      "persistentvolume_size_bytes",
```

Comment:
- I guess `s/size/capacity/' the users can know it's the size of capacity
- better break down by mode, see [PersistentVolumeDiscoveryTotal](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/6632c7dc895f11318747663a519aa400e6d96fc7/pkg/metrics/metrics.go#L42-L49) as an example, then end-users can know the total capacity in size for all volumes types (by summing) and also for each mode

Reply:
I've done the requests changes 👍

Reply:
LGTM, one last nit: update metrics docs [here](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/provisioner.md#metrics)